internal/services/recommender.go

```diff
@@ -105,7 +105,7 @@ func ProcessEvent(msg *kafka.Message) {
 		if err != nil {
 			log.Errorf("Unable convert list_of_experiments to json: %s", err)
 		}
-		_ = p.SendMessage(msgBytes, &cfg.ExperimentsTopic)
+		_ = p.SendMessage(msgBytes, &cfg.ExperimentsTopic, kafkaMsg.Kafka_request_msg.Metadata.Org_id)
```

Comment:
Shall we include the `request_id` as well along with the `org_id`, like so <org_id>-<request_id>?

Reply:
I think we should not add it considering a scenario when customer is in disconnected env and manually uploading 4 reports to consoledot in chronological order.